src/wdf.cpp

```diff
@@ -278,3 +359,48 @@ WDF_DRIVER_CONFIG_INIT(_Out_ PWDF_DRIVER_CONFIG config, _In_opt_ PFN_WDF_DRIVER_
 {
     *config = {.EvtDriverDeviceAdd = evt_driver_device_add};
 }
+
+HANDLE
+usersim_get_device_handle(HMODULE module)
```

Comment:
Drivers can create multiple device objects, at least in the WDM model. Should this routine take a device name as a parameter?

Reply:
Updated prototype and filed #114 to track actual support